extensions/mongodb/src/main/java/com/hazelcast/jet/mongodb/MongoSinkBuilder.java

```diff
@@ -244,7 +245,22 @@ public MongoSinkBuilder<T> writeMode(@Nonnull WriteMode writeMode) {
      */
     @Nonnull
     public MongoSinkBuilder<T> throwOnNonExisting(boolean throwOnNonExisting) {
-        params.setThrowOnNonExisting(throwOnNonExisting);
+        params.setCheckExistenceOnEachConnect(throwOnNonExisting);
```

Comment:
`throwOnNonExisting` does not work, it is overwritten in `MongoSinkBuilder.build()` based on `existenceChecks`

Reply:
Right, leftover. I removed it.